addOns/pscanrules/CHANGELOG.md

```diff
@@ -10,6 +10,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Update reference for X-Content-Type-Options Header Missing and Content-Type Header Missing (Issue 8262).
     - They now also include example alerts for documentation generation and cross linking purposes (Issues 6119, 7100, and 8189).
 - Update reference for Loosely Scoped Cookie (Issue 8262).
+- On Non-LOW threshold, PII Scan rule only evaluates HTML, JSON and XML responses(Issue 8264).
+
+### Fixed
+- PII Scan rule no longer reports PII disclosure on javascript files on MEDIUM and HIGH threshold.
```

Comment:
`JavaScript`

Reply:
I intentionally removed it assuming the changelog added above( `On Non-LOW threshold....`) will convey the same message.

Reply:
Do I add it again? If yes below the `non low` change log or under a separate heading `fixed`

Reply:
Ok, leave as is then.